src/components/amount/amount.tsx

```diff
@@ -37,6 +38,24 @@ export const AmountComponent = (props: IExternalProps & IReduxProps) => {
         props.tokenDecimals
     );
 
+    if (props.smallFontToken) {
```

Comment:
this is not quite ok, just because you need to display a different text color for the currency we have basically 2 implementaion based on `smallFontToken` flag. Loose the IF and make the component more customisable.